src/server/BuildableAutomationServer.ts

```diff
@@ -214,13 +214,8 @@ export class BuildableAutomationServer extends AbstractAutomationServer {
     }
 
     private enrichContext(ctx: HandlerContext): HandlerContext {
-        return {
-            teamId: ctx.teamId,
-            correlationId: ctx.correlationId,
-            invocationId: ctx.invocationId,
-            messageClient: ctx.messageClient,
-            graphClient: ctx.graphClient ? ctx.graphClient : this.graphClient,
-        };
+        ctx.graphClient = ctx.graphClient || this.graphClient;
+        return ctx;
```

Comment:
Same question I had on @jessitron's PR: is it safe to change the semantics here to modify the passed object instead of creating a new one? Plus, now that you've added the lifecycle property, should we be dealing with that here too?

Reply:
@jessitron's implementation is correct. The old implementation already lost the `context` attribute. With her change we are safe. That's good.